reactComponents/.storybook/main.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
WELL DONE SIR!

---
By the way, I added storybook to the package.json so we don't need it installed globally to run